DUI3-DX/DUI3/Speckle.Connectors.DUI/Bindings/SendBindingUICommands.cs

```diff
@@ -12,6 +12,8 @@ public class SendBindingUICommands
 
   private IBridge _bridge;
 
+  public delegate SendBindingUICommands Factory(IBridge bridge);
```

Comment:
Not sure if we need this 🤔

Reply:
I am currently using this in the RhinoSendBinding constructor to create the sendBindingUiCommands with the same instance of the IBridge as the SendBinding. We could remove it one of two ways,
1. Inject Func<IBridge, SendBindingUiCommands> instead of this factory
2. Create a new container scope for each IBinding and then register the IBridge as scoped.